src/graph/backend/dnnl/fusion_info.cpp

```diff
@@ -49,7 +49,7 @@ dnnl::primitive_attr make_dnnl_primitive_attr(
 
     if (fusion_info.dst_scales_) {
         const op_t *dst_scales_op = fusion_info.dst_scales_->get_op();
-        assertm(fusion_info.with_runtime_scales(false, 0),
+        VCHECK_FUSION_INFO(fusion_info.with_runtime_scales(false, 0), attr,
                 "only support runtime dst scales.\n");
```

Comment:
random spot: do we really need the `\n` in the verbose string? Will it cause format issue in verbose log?

Reply:
thanks . I removed all `\n` in macro.
`\n` will cause extra empty line in verbose log, it already has one.

```
#define VFORMAT(stamp, flagkind, apitype, logtype, logsubtype, msg, ...) \
    do { \
        std::string stamp_; \
        if (dnnl::impl::get_verbose_timestamp()) \
            stamp_ = std::to_string(stamp) + ","; \
        dnnl::impl::verbose_printf(flagkind, \
                "%s" CONCAT2(VERBOSE_, apitype) "," CONCAT2( \
                        VERBOSE_, logtype) "%s," msg "\n", \
                stamp_.c_str(), logsubtype, ##__VA_ARGS__); \
    } while (0)
```